gcp/README.md

```diff
@@ -42,6 +42,16 @@ The DNS zones are:
 
 The environments that run in GCP need some initial resources that must be created by an administrator first. The [common part of this repository](../common) has the code and the instructions to do so.
 
+## Permissions
+
+The permissions in this project are set in three different levels: at organization level, at project level and at resource level.
+
+At the organization level we have the group _cloud-admin@raisingthefloor.org_ which contains the list of users that will manage the projects of the organization and has the high level permissions to do so. Also we have a SA dedicated for the project creation and the billing association: _projectowner@gpii-common-prd.iam.gserviceaccount.com_. This SA only has the enough permissions to create projects, assciate them to the billing account and create the IAMs needed in such project to allow the owner to create the resources inside it. The SA _projectowner@gpii2test-common-stg.iam.gserviceaccount.com_ must also be in the organization level, as the billing account is associated to this organization and it is used to attach it to the testing organization _test1.gpii.net_.
```

Comment:
Does this new arrangement require any changes to our team onboarding documentation?

Reply:
It's done in the onboarding and offboarding documentation.